Assets/Scripts/Controller2.cs

```diff
@@ -34,7 +35,9 @@ public class Controller2 : MonoBehaviour {
     }
 
 	void Update(){
-		manageInput ();
+        if (allowInput) {
```

Comment:
Note: This was added because the controller 2 that was attached to the human doll was canceling the A button press (if you look at the manageInput funciton there is an Input.ResetAxises() call). Also this way the cancel ghost mode sound doesn't play when you press A.